src/glass/src/app/core/dashboard/widgets/services-capacity-dashboard-widget/services-capacity-dashboard-widget.component.html

```diff
@@ -0,0 +1,17 @@
+<glass-widget [loadData]="loadData.bind(this)"
```

Comment:
Unfortunately another Chrome (Version 89.0.4389.82) issue. It doesn't happen on Firefox.

![screen-capture](https://user-images.githubusercontent.com/8761082/113147252-e3a10c00-9230-11eb-9d78-9dc937c7c016.gif)

Reply:
I don't see that happen in chrome or chromium on Ubuntu 18.04. Maybe distro related?